### ISSUE TITLE
Import option to add a RESET animation

### ISSUE THREAD

**Issue by sim-kaart** [viniciusgerevini/godot-aseprite-wizard#171]
It is common practice for us to have a RESET animation in our animation players. I could add one to every Aseprite file, so that the wizard automatically generates it, but it would be cleaner to have it as an import option.

**Comment by viniciusgerevini** [comment]
Yeah, that makes sense. I reckon the plugin can use any RESET animation set in aseprite itself, and fallback to the first frame from the first animation in case a RESET animation is not present. Was that what you had in mind?

**Comment by sim-kaart** [comment]
Yes that’d work! Hadn’t thought about the ‘preset RESET’ needing a frame. And you’re right, first frame of first animation makes sense. Thanks :)

**Comment by KeyboardDanni** [comment]
Whenever I reimport my sprites due to incremental changes, I have to recreate the RESET value because the plugin keeps removing it, so this would be nice to have.

I hadn't thought about just putting a RESET frame in the Aseprite file itself. Have you run into any issues with doing things this way? I might do this myself, though I'd prefer if the plugin just defaulted to making RESET put the sprite on frame 0.

**Comment by viniciusgerevini** [comment]
Thanks for weighting in. Yes, I have used the RESET tag in aseprite before and it works, but I agree it's more convenient if the plugin does that automatically.
